test/scenarios/ynEIGEN/ynLSDeWithdrawals.t.sol

```diff
@@ -26,109 +26,36 @@ contract ynLSDeWithdrawalsTest is ynLSDeScenarioBaseTest {
-        }
-
-        // deploy WithdrawalQueueManager
-        {
-            TransparentUpgradeableProxy _proxy = new TransparentUpgradeableProxy(
-                address(new WithdrawalQueueManager()),
-                actors.admin.PROXY_ADMIN_OWNER,
-                ""
-            );
-            withdrawalQueueManager = WithdrawalQueueManager(address(_proxy));
-        }
-
-        // deploy wrapper
-        {
-            // call `initialize` on LSDWrapper
-            TransparentUpgradeableProxy _proxy = new TransparentUpgradeableProxy(
-                address(
-                    new LSDWrapper(
-                        chainAddresses.lsd.WSTETH_ADDRESS,
-                        chainAddresses.lsd.WOETH_ADDRESS,
-                        chainAddresses.lsd.OETH_ADDRESS,
-                        chainAddresses.lsd.STETH_ADDRESS)
-                    ),
-                actors.admin.PROXY_ADMIN_OWNER,
-                abi.encodeWithSignature("initialize()")
-            );
-            wrapper = LSDWrapper(address(_proxy));
-        }
-
-        // initialize RedemptionAssetsVault
         {
-            RedemptionAssetsVault.Init memory _init = RedemptionAssetsVault.Init({
-                admin: actors.admin.PROXY_ADMIN_OWNER,
-                redeemer: address(withdrawalQueueManager),
-                ynEigen: yneigen,
-                assetRegistry: assetRegistry
-            });
-            redemptionAssetsVault.initialize(_init);
-        }
-
-        // initialize WithdrawalQueueManager
-        {
-            WithdrawalQueueManager.Init memory _init = WithdrawalQueueManager.Init({
-                name: "ynLSDe Withdrawal Manager",
-                symbol: "ynLSDeWM",
-                redeemableAsset: IRedeemableAsset(address(yneigen)),
-                redemptionAssetsVault: redemptionAssetsVault,
-                admin: actors.admin.PROXY_ADMIN_OWNER,
-                withdrawalQueueAdmin: actors.ops.WITHDRAWAL_MANAGER,
-                redemptionAssetWithdrawer: actors.ops.REDEMPTION_ASSET_WITHDRAWER,
-                requestFinalizer:  actors.ops.REQUEST_FINALIZER,
-                withdrawalFee: 0,
-                feeReceiver: actors.admin.FEE_RECEIVER
-            });
-            withdrawalQueueManager.initialize(_init);
-        }
-
-        // upgrade and initialize:
-        // (1) tokenStakingNode implementation
-        // (2) ynLSDe
-        // (3) EigenStrategyManager
-        // (4) AssetRegistry
-        // (5) TokenStakingNodesManager
-        // (6) ynEigenDepositAdapter
-        _upgradeContracts();
-
-        // unpause transfers
-        {
-            vm.prank(actors.admin.UNPAUSE_ADMIN);
-            yneigen.unpauseTransfers();
-        }
-
-        // grant burner role
-        {
-            vm.startPrank(actors.admin.STAKING_ADMIN);
-            yneigen.grantRole(yneigen.BURNER_ROLE(), address(withdrawalQueueManager));
+            // Get pending redemption amount
```

Comment:
this just fulfills for existing pending - not perfect but with ever changing state this is ok.

We need to test more in detail in integration tests with fresh deploys.